apps/masterbots.ai/components/routes/chat/chat-message.tsx

```
@@ -71,72 +47,64 @@ export function ChatMessage({
                   {...props}
                 >
                   {children}
                 </a>
               )
             },
-            code({ inline, className, children, ...props }) {
+            code({ node, inline, className, children, ...props }) {
+              //* Handle cursor animation for streaming responses
               if (children.length) {
-                if (children[0] == '▍') {
+                if (children[0] == '▍')
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)